enhancements/proposals/1978-scylladbdatacenter-api/README.md

```diff
@@ -0,0 +1,764 @@
+  and discouraged by not having a field to set it in API. [#1378](https://github.com/scylladb/scylla-operator/issues/1378)
+  To preserve backward compatibility controllers will support enabling it by setting special 
+  annotation `internal.scylla-operator.scylladb.com/enable-host-networking` to `true` on `v1alpha1.ScyllaDBDatacenter`.
+  The controller migrating from `v1.ScyllaCluster` to `v1alpha1.ScyllaDBDatacenter` will set this annotation when required.
+- `sysctls` doesn't belong to ScyllaDB cluster nor datacenter, these are configuration options for nodes. 
+  A better place to have them is NodeConfig. Until it's available the NodeConfig, we will support setting them 
+  via `internal.scylla-operator.scylladb.com/set-sysctls` annotation, in a format `{key}={value},{key}={value}`.
+
+### Helm chart
+
+The existing `scylla` helm chart will use the existing `v1.ScyllaCluster`. It has several differences between what's in CRD
+and what's in the chart templates, hence it cannot be simply mapped one-to-one.
+A new helm chart called `scylladb-datacenter` will simply copy `spec` from `values` to a new CRD solving this issue and making it
+easily extensible, while keeping the schema of CRD and helm schema in sync.
+
+### Scylla Manager integration
+
+Scylla Manager tasks do not fit into a single datacenter, as Manager tasks work on the entire cluster.
+These are not yet transferred from the spec and status of the old API until a better alternative for these is available.
+The existing controller reconciling these will continue using `v1.ScyllaCluster`, users wanting to use these,
+won't be able to transition to the new API until an extension is available.
+Extending the manager integration may be addressed in a future proposal that rethinks the overall approach in more detail.
+
+### Topology placement
+
+Currently, to provide placement rules for a rack, users have to provide a very lengthy `nodeAffinity`.
+Even the simplest rules require providing 10'ish lines with 4'ish indentation levels.
+This makes the entire resource huge and UX isn't great.
+To improve it, the new field `topologyLabelSelector` added on the datacenter and rack level,
+provides a label selector which will be translated into placement nodeAffinity rules selecting nodes matching these labels.
+Label selectors from the datacenter and rack level will be merged into the final node affinity. 
+In case of a collision on keys, rack ones take precedence.
+
+To prevent misuse of this simplification keys of the provided label selector will be validated against the `topology.kubernetes.io/*` rule,
+and usage of both `nodeAffinity` and `topologyLabelSelector` will be forbidden by the webhook.
+Users wanting to construct more complex placement rules will have to do it using `placement` spec.
+
+Given following spec:
+```yaml
+spec:
+  topologyLabelSelector:
+    topology.kubernetes.io/region: us-east1
```

Comment:
nit: region is in most cases identical for all nodes in the cluster and is already selected by the zone. At this level I'd primarily set node pool labels selector as an example.

Reply:
done